lib/presentation/pages/whish_list/widget/whish_list_widget.dart

```diff
@@ -0,0 +1,233 @@
+import 'package:flutter/material.dart';
+import 'package:furniture_shop/presentation/widgets/base/custom_text.dart';
+import 'package:furniture_shop/presentation/widgets/primary_button.dart';
+import 'package:furniture_shop/values/colors.dart';
+import 'package:furniture_shop/values/dimens.dart';
+import 'package:furniture_shop/values/font_sizes.dart';
+
+import 'icon_whishlist_widget.dart';
+
+class WhishListWidget extends StatelessWidget {
+  final String? title;
+  final String? imageUrl;
+  final double? price;
+  final bool? checked;
+  final int? statusWhish;
+
+  const WhishListWidget({
+    this.title,
+    this.imageUrl,
+    this.price,
+    this.checked = false,
+    this.statusWhish = 0, //0: In stock ---- 1: Sold out
+    Key? key,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      margin: const EdgeInsets.symmetric(vertical: AppDimen.spacing_1),
+      child: Row(
+        children: [
+          _buildRadioCheckbox(),
+          _buildImage(),
+          _buildInfo(),
+          Spacer(),
+          _buildButtonMore(context),
+        ],
+      ),
+    );
+  }
+
+  Widget _buildRadioCheckbox() {
+    return Container(
+      decoration: BoxDecoration(
+        shape: BoxShape.circle,
+        color: statusWhish == 1 ? Colors.white : Colors.black,
+        border: Border.all(
+          width: 1,
+          color: statusWhish == 1 ? AppColor.colorGrey : Colors.transparent,
+        ),
+      ),
+      child: Padding(
+        padding: const EdgeInsets.all(3.0),
+        child: Icon(
```

Comment:
Change to IconButton (Tap to choose product)

Reply:
done